src/Execution/lbsRuntime.py

```diff
@@ -76,13 +78,23 @@ def __init__(self, p, c: dict, order_strategy: str, v=False):
         else:
             self.phase = p
 
-        # Verbosity of runtime
-        self.verbose = v
+        # Instantiate work model
+        self.work_model = WorkModelBase.factory(
+            w.get("name"),
+            w.get("parameters", {}))
+        if not self.work_model:
+            print(bcolors.ERR
```

Comment:
So, maybe we switch to fstring with every new commit :)?

Reply:
Better yet we should have a helper function called `print_error()`, another one called `print_warning()` and yet another one called `print_message()` that would generically deal with the properly-colored headings. In addition I would make the warnings have an orange-colored heading, the regular messages have a green heading (and keep the red for errors).